pkg/util/util.go

```diff
@@ -451,3 +451,12 @@ func ObjectManagedByClusterRegistry(object metav1.Object) bool {
 	_, ok := annotations[clusterregv1alpha1.OwnershipAnnotation]
 	return ok
 }
+
+// TruncatedCommonName ensures that the passed-in common name doesn't exceed the specified number of characters
+func TruncateStringByLen(name string, l int) string {
```

Comment:
Is it enough just to simply truncate the common name or we have to ensure that it stays unique after truncation? Eg two different KafkaUsers should be represented by two different certificates.

If uniqueness needs to be ensured than a has code could be be computed from the original value and appended to the truncated value.

Reply:
I thought about hashing the entire string and limiting it to 64 chars as well, but I realized it could be difficult for the users to debug if things go wrong since the hashed string is un-comprehensible.

In terms of uniqueness, since the common name is constructed with `cluster.Name` and`cluster.Namespace`, I thought this is enough to ensure the uniqueness of the entire string since we cant't have the same kafka cluster name under the same namespace